python/cucim/src/cucim/skimage/_shared/filters.py

```diff
@@ -133,21 +112,7 @@ def gaussian(
     >>> from skimage.data import astronaut
     >>> image = cp.array(astronaut())
     >>> filtered_img = gaussian(image, sigma=1, channel_axis=-1)
-
     """
-    if channel_axis is ChannelAxisNotSet:
```

Comment:
Do we still need to think about the `ChannelAxisNotSet` case?

Reply:
No that was a temporary class used during the deprecation of old behavior that tried to guess if it was an RGB image based on the size of the last axis. The new default is that the user has to explicitly specify if that want an axis to be treated as channels.